server/configs/certs/server/server-cert.pem

```diff
@@ -0,0 +1,34 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
It would be nice to document the commands or provide a script for generating all of the test certs for when these occasionally need to get updated.

Reply:
I added the script and a small note in `server/configs/cert`. Do you think that is enough ?